acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ThreadPantheonNodeRunner.java

```diff
@@ -93,7 +94,7 @@ public void startNode(final PantheonNode node) {
             .bannedNodeIds(Collections.emptySet())
             .metricsSystem(noOpMetricsSystem)
             .metricsConfiguration(node.metricsConfiguration())
-            .permissioningConfiguration(node.getPermissioningConfiguration())
+            .permissioningConfiguration(Optional.of(node.getPermissioningConfiguration()))
```

Comment:
Prefer to see the RunnerBuilder default to having an empty permissioning config and this setter be passed a PermissioningConfig without the Optional and pivot whether we call this based on whether we have a config or not.

Reply:
A good suggestion.  Will update.